### ISSUE TITLE
TensorBoard: Embeddings visualization not supported yet

### ISSUE THREAD

**Issue by Yannik1337** [wandb/client#1516]
For a project, I create some .tsv files. They are saved within the TensorBoard dir. When this directory is e.g. /tensorboard/run20201711/, the files are saved under this directory. The attached screenshot shows such logging structure.

![tree-view dir](https://user-images.githubusercontent.com/50734087/99505549-416b4080-2981-11eb-8b13-2c2c6be08b9c.png)

I then check to see if I can visualize these files as embedding files (similar to [projector.tensorflow.org](projector.tensorflow.org)), in the TensorBoard tab on my W&B panel.
However, it greets me with no data found - all other metrics are logging fine, images and so are uploaded correctly.

From your documentation at [https://docs.wandb.com/library/log](https://docs.wandb.com/library/log), I found no way to manually log the files, since .tsv/.csv do not seem to be supported.

How can I sync the files to my W&B project?

**Comment by vanpelt** [comment]
You can use `wandb.save`: https://docs.wandb.com/library/save or just write the files into `wandb.run.dir`.

**Comment by Yannik1337** [comment]
Thanks, this solves my problem. Did not know about this feature!

**Comment by Yannik1337** [comment]
Edit: While W&B does now successfully sync the files to cloud, when using TensorBoard via your web interface, I can't load the embeddings.

Following structure is present after the run:
![tree](https://user-images.githubusercontent.com/50734087/99632089-ef87f080-2a3c-11eb-8d2c-a69e3900a162.png)

And this is the screen it greets me with when checking the embeddings:
![embeddings](https://user-images.githubusercontent.com/50734087/99632211-22ca7f80-2a3d-11eb-9e52-ed04666e81fa.png)

**Comment by ariG23498** [comment]
Hey @YanickSchraner 
I tried reproducing your issue.

I [followed this tutorial](https://www.tensorflow.org/tensorboard/tensorboard_projector_plugin) and made the `tsv` files.
While following the tutorial, I have changed slightly and used `logs/` as the directiry where the checkpoints and tsv files are created. Due to the fact that wandb tensorboard get triggered only in the presence of tfevents, I went out and added a few lines of code and creates some `scalar tfevents`
```python
writer = SummaryWriter("logs")
scalars = {
    'scalar_1': 1.0,
    'scalar_2': 2.0,
    'scalar_3': 3.0
}
writer.add_scalars('test', scalars, global_step=1)
writer.close()
```

Later I have used the following scrip to sync the files to wandb
```python
wandb.init(entity='xxx', project='xxx')
wandb.save('logs/*', base_path="logs/")
wandb.finish()
```

This has the following files synced
![image](https://user-images.githubusercontent.com/36856589/101372137-92f85280-38d1-11eb-890a-09d63d68952f.png)

With this in place, the tensorboard projector seems to be working like a charm
![image](https://user-images.githubusercontent.com/36856589/101372383-d783ee00-38d1-11eb-848d-ba2dc010e193.png)


I think the problem here is the checkpoint creation and the file structure. I think having all the files in the root directory is easier for  tensorboard to locate.

**Comment by adricarda** [comment]
Still not working for me, while on a local version of Tensorboard everything works fine. Any updates on this?

**Comment by ariG23498** [comment]
Hey @adricarda 
Could you describe your issue a little better?

**Comment by adricarda** [comment]
Sure, sorry for the delay.
What I mean is that when generating an embedding projection with writer.add_embedding() (I am using pytorch and torch.utils.tensorboard) I can launch locally tensorboard and use the projector correctly. 
All files are written in the "files" directory automatically created by wandb for the current run, so that everything is pushed on the wandb server as well.
On the other hand, when I use the tensorboard tab from the wandb web page (as showed above) I can see everything that I logged (i.e. lr, loss, metrics...) but the projector tab shows a blank page.

**Comment by ariG23498** [comment]
That is interesting
Would you like to share a colab notebook for us to quickly review the whole thing and revert back to you?

**Comment by adricarda** [comment]
Yes, I cannot directly share my code but I will try to make a minimal reproducible example in the following days. I'll keep you posted.

**Comment by sydholl** [comment]
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.

**Comment by ysig** [comment]
@sydholl Still doesn't work correctly...
Here is a [MRE](https://colab.research.google.com/drive/1-Ah27rXgzF37YOVsMXIg2nmmoCAR_UrM?usp=sharing)
I think if you can copy recursively the produced file structure under tensorboard logs inside wandb files it would work but wandb.save doesn't allow me to do so.
